marklogic-client-api/src/main/java/com/marklogic/client/type/PlanSearchOptions.java

```diff
@@ -15,19 +15,19 @@
  */
 package com.marklogic.client.type;
 
-// IMPORTANT: Do not edit. This file is generated.
-
 /**
  * An option controlling the scoring and weighting of fromSearch()
  * for a row pipeline.
  */
 public interface PlanSearchOptions {
-    XsDoubleVal getQualityWeight();
+	XsFloatVal getQualityWeight();
     ScoreMethod getScoreMethod();
-    PlanSearchOptions withQualityWeight(double qualityWeight);
-    PlanSearchOptions withQualityWeight(XsDoubleVal qualityWeight);
+	XsDoubleVal getBm25LengthWeight();
```

Comment:
For each new method in this, add the following javadoc:

```
/**
 * @since 6.7.0
 */
```

We haven't been good with that in the Optic plan builder stuff, but it's mostly been added everywhere else in the Java Client since 6.0.